packages/apps/explorer/.eslintrc.js

```diff
@@ -3,6 +3,14 @@
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  extends: ['@kadena/eslint-config/profile/lib'],
+  extends: [
+    '@kadena/eslint-config/profile/lib',
+    'eslint-config-airbnb-typescript',
```

Comment:
I think this conflicts with the rules we have from the rig. I think this can be removed (might result in a bunch of errors that need to be fixed)